### PR TITLE
TCC review

### PR DESCRIPTION
**Pull request by sherenewso2**
## Purpose
TCC review of descriptions and code comments of samples.

### CODE REVIEW

**Comment by sherenewso2** [comment]
@dulanjal please review.

File: stdlib/ballerina-database/src/main/ballerina/jdbc/Package.md [hunk 1]
```diff
@@ -232,7 +232,7 @@ match ret9 {
 
 ### Get proxy table
 
-A proxy for a database table that allows performing add/remove operations over the actual database table, can be obtained by `getProxyTable` action as follows.
+Use the `getProxyTable` action shows below to obain a proxy for a database table that allows performing add/remove operations over the actual database table.
```
Comment (appellj) [inline stdlib/ballerina-database/src/main/ballerina/jdbc/Package.md]: Change "shows below" to "shown below"
Change "obain" to "obtain"

Comment (sherenewso2) [reply stdlib/ballerina-database/src/main/ballerina/jdbc/Package.md]: Fixed the typos. Thanks!

File: stdlib/ballerina-database/src/main/ballerina/jdbc/Package.md [hunk 2]
```diff
@@ -24,14 +24,14 @@ endpoint jdbc:Client testDB {
     dbOptions: { useSSL: false }
 };
 ```
-The full list of endpoint properties can be found in the `sql:PoolOptions` type.
+The full list of endpoint properties can be found listed under the `sql:PoolOptions` type located in the `types.bal` file of the SQL package directory.
```
Comment (lafernando) [inline stdlib/ballerina-database/src/main/ballerina/jdbc/Package.md]: Don't we need a comma after "type" here?

File: stdlib/ballerina-database/src/main/ballerina/jdbc/Package.md [hunk 3]
```diff
@@ -24,14 +24,14 @@ endpoint jdbc:Client testDB {
...
 
 ### Creating tables
 
-Following is an example of creating a table with two columns. One field is of type int and the other of varchar. The CREATE statement is executed via the `update` operation of the endpoint.
+This sample creates a table with two columns. One field is of type `int` and the other of is of type `varchar`. The CREATE statement is executed via the `update` operation of the endpoint.
```
Comment (lafernando) [inline stdlib/ballerina-database/src/main/ballerina/jdbc/Package.md]: Comma after `int`?

File: stdlib/ballerina-database/src/main/ballerina/jdbc/Package.md [hunk 4]
```diff
@@ -24,14 +24,14 @@ endpoint jdbc:Client testDB {
...
 
 ```ballerina
-//Create ‘Students’ table with fields ‘StudentID’ and ‘LastName’.
+//Create the ‘Students’ table with fields ‘StudentID’ and ‘LastName’.
```
Comment (lafernando) [inline stdlib/ballerina-database/src/main/ballerina/jdbc/Package.md]: Please put a space after "//". Fix in all places.

File: stdlib/ballerina-database/src/main/ballerina/jdbc/Package.md [hunk 5]
```diff
@@ -77,9 +77,9 @@ match ret3 {
 }
 ```
 
-### Inserting with Returning generated keys
+### Inserting data with auto-generated keys
 
-Following is an example of inserting data, while returning the auto generated keyes by executing an INSERT statement using the `updateWithGeneratedKeys` operation.
+This example demonstrates inserting data while returning the auto-generated keys by executing an INSERT statement that uses the `updateWithGeneratedKeys` operation.
```
Comment (lafernando) [inline stdlib/ballerina-database/src/main/ballerina/jdbc/Package.md]: Please re-evaluate the usage of commas.

File: stdlib/ballerina-database/src/main/ballerina/jdbc/Package.md [hunk 6]
```diff
@@ -153,15 +153,15 @@ match ret4 {
 
 ### Batch updating data
 
-Following example demonstrates how to insert multiple records with a single INSERT statement executed via the `batchUpdate` operation of the endpoint. This is done by first creating multiple parameter arrays, each representing a single record, and then providing those to the `batchUpdate` operation. Similarly, multiple UPDATE statements could be also executed via `batchUpdate`.
+This example demonstrates how to insert multiple records with a single INSERT statement that is executed via the `batchUpdate` operation of the endpoint. This is done by first creating multiple parameter arrays, each representing a single record, and then passing those arrays to the `batchUpdate` operation. Similarly, multiple UPDATE statements can also be executed via `batchUpdate`.
```
Comment (lafernando) [inline stdlib/ballerina-database/src/main/ballerina/jdbc/Package.md]: Please re-evaluate the usage of commas.

File: stdlib/ballerina-database/src/main/ballerina/jdbc/Package.md [hunk 7]
```diff
@@ -179,9 +179,9 @@ match ret5 {
 
 ### Calling stored procedures
 
-Following are three examples of executing stored procedures via the  `call` operation of the endpoint. 
+The following examples demonstrate executing stored procedures via the  `call` operation of the endpoint.
```
Comment (lafernando) [inline stdlib/ballerina-database/src/main/ballerina/jdbc/Package.md]: There is an extra space before `call`.

File: stdlib/ballerina-database/src/main/ballerina/jdbc/Package.md (hunk 2 above)
Comment (appellj) [reply stdlib/ballerina-database/src/main/ballerina/jdbc/Package.md]: No, unless you rewrite it like this: "...under the `sql:PoolOptions` type, which is located in the `types.bal` file..."

File: stdlib/ballerina-database/src/main/ballerina/jdbc/Package.md (hunk 3 above)
Comment (appellj) [reply stdlib/ballerina-database/src/main/ballerina/jdbc/Package.md]: Yes, insert comma after `int`

File: stdlib/ballerina-database/src/main/ballerina/jdbc/Package.md (hunk 5 above)
Comment (appellj) [reply stdlib/ballerina-database/src/main/ballerina/jdbc/Package.md]: No comma, but you might want to break this up into two sentences to make it easier to parse. "This example demonstrates inserting data while returning the auto-generated keys. It achieves this by executing an INSERT statement that uses the `updateWithGeneratedKeys` operation."

Comment (sherenewso2) [reply stdlib/ballerina-database/src/main/ballerina/jdbc/Package.md]: I think this sentence makes more sense if we replace the word "while" with "and":

This example demonstrates inserting data **and** returning the auto-generated keys by executing an INSERT statement that uses the `updateWithGeneratedKeys` operation. Is this okay?

Comment (appellj) [reply stdlib/ballerina-database/src/main/ballerina/jdbc/Package.md]: Actually, I'm wondering if the meaning is correct. Does the INSERT statement use the operation, or does the operation execute the INSERT statement? If the latter, I'd rewrite this as "This example demonstrates inserting data while returning the auto-generated keys. It achieves this by using the `updateWithGeneratedKeys` operation to execute the INSERT statement."

File: stdlib/ballerina-database/src/main/ballerina/jdbc/Package.md (hunk 6 above)
Comment (appellj) [reply stdlib/ballerina-database/src/main/ballerina/jdbc/Package.md]: The commas look fine.

File: stdlib/ballerina-database/src/main/ballerina/jdbc/Package.md (hunk 5 above)
Comment (sherenewso2) [reply stdlib/ballerina-database/src/main/ballerina/jdbc/Package.md]: @dulanjal could you please clarify which meaning we want to give in this sentence?

File: stdlib/ballerina-database/src/main/ballerina/jdbc/Package.md (hunk 3 above)
Comment (sherenewso2) [reply stdlib/ballerina-database/src/main/ballerina/jdbc/Package.md]: Fixed with this commit: https://github.com/ballerina-platform/ballerina-lang/pull/8071/commits/b100133f6879d70445f59461cd7a1c84ecc1c05b

File: stdlib/ballerina-database/src/main/ballerina/jdbc/Package.md (hunk 4 above)
Comment (sherenewso2) [reply stdlib/ballerina-database/src/main/ballerina/jdbc/Package.md]: Fixed with this commit: https://github.com/ballerina-platform/ballerina-lang/pull/8071/commits/b100133f6879d70445f59461cd7a1c84ecc1c05b

File: stdlib/ballerina-database/src/main/ballerina/jdbc/Package.md (hunk 5 above)
Comment (sherenewso2) [reply stdlib/ballerina-database/src/main/ballerina/jdbc/Package.md]: Checked with @dulanjal. Yes, the operation executes the INSERT statement. Changed this sentence as suggested by @appellj with this commit: https://github.com/ballerina-platform/ballerina-lang/pull/8071/commits/b100133f6879d70445f59461cd7a1c84ecc1c05b

File: stdlib/ballerina-database/src/main/ballerina/jdbc/Package.md (hunk 7 above)
Comment (sherenewso2) [reply stdlib/ballerina-database/src/main/ballerina/jdbc/Package.md]: Fixed with this commit: https://github.com/ballerina-platform/ballerina-lang/pull/8071/commits/b100133f6879d70445f59461cd7a1c84ecc1c05b

File: stdlib/ballerina-database/src/main/ballerina/jdbc/Package.md (hunk 2 above)
Comment (lafernando) [reply stdlib/ballerina-database/src/main/ballerina/jdbc/Package.md]: I see, thanks. I felt your version is easier to read, than a long one sentence that doesn't break anywhere.

File: stdlib/ballerina-database/src/main/ballerina/jdbc/Package.md (hunk 6 above)
Comment (lafernando) [reply stdlib/ballerina-database/src/main/ballerina/jdbc/Package.md]: I thought you need a comman before "that is", I guess  you don't.

Comment (appellj) [reply stdlib/ballerina-database/src/main/ballerina/jdbc/Package.md]: No, you would use a comma before "which", not "that"

Comment (lafernando) [reply stdlib/ballerina-database/src/main/ballerina/jdbc/Package.md]: Got it, thanks.

**Comment by lafernando** [comment]
Please fix the conflicts.

**Comment by sherenewso2** [comment]
@lafernando Fixed the conflicts now

**Pull request merged**